mailer/__main__.py

```diff
@@ -0,0 +1,33 @@
+import time
+
+from .daemon import Daemon
+from .message import UserEmail
+
+
+def main():
+    daemon = Daemon()
+    while True:
+        try:
+            daemon.tick()
+            mail_file = daemon.get_email_file()
+            if not mail_file:
+                time.sleep(5)
+                continue
+
+            email = UserEmail.from_file(mail_file)
+            net = daemon.get_network(email.user)
+            if not net:
+                daemon.error_reporter.report_warning(
+                    "Unable to determine network for user {!r}".format(email.user)
+                )
+                continue
+
+            net.send_email(email)
+            mail_file.unlink()
```

Comment:
Possibly a decent idea to zero the file a few times first? given it may contain userdata, or password reset tokens etc.

Reply:
That is entirely ineffective on any server running on an SSD. I can truncate the file first but that's about it.

Reply:
There is no guarantee that this will always be running on an SSD. calling truncate() would do nothing really to solve the question of left over data, as it just resizes the file.

Reply:
Due to the discussion on IRC about using `/var/run`, this is no longer relevant.